### PR TITLE
add copy draft for 2023-05-10

### PR DESCRIPTION
**Pull request by patrick-j-govalpha**
First draft of copy.

Some TBDs:

* smart contract details for RWA014 - pending merge on Goerli
* Clarify whether or not we're including the DAO Resolution for TACO

### CODE REVIEW

File: governance/votes/Executive vote - May 10, 2023.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes and additions** will occur within the Maker Protocol:
+- RWA014 (Coinbase Custody) will be onboarded as a new vault type, as detailed below.
+- Multiple risk parameters for decentralized collateral types will be updated, as detailed below.
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - May 10, 2023.md:20]: Is it right to refer to WBTC as decentralized anymore?

File: governance/votes/Executive vote - May 10, 2023.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+- The Starknet Bridge Limit will be increased to 5 million DAI, as detailed below.
+- DAI and MKR streams will be set up for PullUp, as detailed below.
+- The old streams for Phoenix Labs will be `yank`ed, and new streams will be created, as detailed below.
+- The `cap` for MKR_VEST will be increased to 2,200 MKR, as detailed below.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. This is to ensure that at least some auction keepers are available to react to unforeseen issues with their bots in the event of early liquidations on new collateral types.
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - May 10, 2023.md:30]:
```suggestion
This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**.
```
Would remove the stuff about liquidation as might cause confusion around coinbase custody.

File: governance/votes/Executive vote - May 10, 2023.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+#### Collateral Parameters
+
+* [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee): **0%**.
+* [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling): **500 million DAI**.
+* [Liquidation Ratio](https://manual.makerdao.com/parameter-index/vault-risk/param-liquidation-ratio): **100%**.
+* Oracle Price: **500 million DAI**.
+* Debt write-off timelock (tau): **0**.
+* Liquidation Process: Will use the established RWA collateral contracts. See [MIP21](https://mips.makerdao.com/mips/details/MIP21) for details.
+
+#### Relevant Smart Contract Information
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - May 10, 2023.md:51]: Are these definitely all relevant?

File: governance/votes/Executive vote - May 10, 2023.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+
+* Increase the WSTETH-A `gap` by 15 million DAI from 15 million DAI to **30 million DAI**.
+* Increase the WSTETH-B `gap` by 15 million DAI from 15 million DAI to **30 million DAI**.
+* Reduce the WBTC-A `gap` by 10 million DAI from 20 million DAI to **10 million DAI**.
+* Reduce the WBTC-B `gap` by 5 million DAI from 10 million DAI to **5 million DAI**.
+* Reduce the WBTC-C `gap` by 10 million DAI from 20 million DAI to **10 million DAI**.
+
+#### [Ceiling Increase Cooldown (`ttl`)](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl) Changes
+
+* Increase the WSTETH-A `ttl` by 21,600 seconds from 21,600 seconds to **43,200 seconds**.
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - May 10, 2023.md:100]: Shouldn't refer to these in seconds. I think it's how it works in the contract, but its unnecessarily hard to parse for governance-facing stuff.

File: governance/votes/Executive vote - May 10, 2023.md [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+
+### Starknet Bridge Limit Increase
+
+As per [this initial poll](https://vote.makerdao.com/polling/QmUnhQZy) and this [forum update](https://forum.makerdao.com/t/april-26th-2023-spell-starknet-bridge-limit/20589), the Starknet Bridge Limit will be increased as follows, if this executive proposal passes.
+
+* The Starknet Bridge Limit will be increased by 4 million DAI from 1 million DAI to **5 million DAI**.
+
+### Vesting Stream Setup
+
+As per [MIP106](https://mips.makerdao.com/mips/details/MIP106), the following DAI and MKR vesting streams will be created if this executive proposal passes:
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - May 10, 2023.md:111]: Worth linking directly to the relevant section? It's quite a long MIP.

File: governance/votes/Executive vote - May 10, 2023.md [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+| Ecosystem Actor Name | MKR Amount | Start Date | End Date | Cliff Date | Destination Address |
+|---|---|---|---|---|---|
+| PullUp Auditor Wallet | 4,000 | 2023-05-01 | 2025-05-01 | 2023-05-01 | [0x868B44e8191A2574334deB8E7efA38910df941FA](https://etherscan.io/address/0x868B44e8191A2574334deB8E7efA38910df941FA) |
+| Phoenix Labs Auditor Wallet | 986.25 | 2023-05-01 | 2024-05-01 | 2023-05-01 | [0x115F76A98C2268DaE6c1421eb6B08e4e1dF525dA](https://etherscan.io/address/0x115F76A98C2268DaE6c1421eb6B08e4e1dF525dA) |
+
+### Cancel Vesting Streams
+
+As part of the above entry, the previous vesting streams to Phoenix Labs will be cancelled and replaced with the new vesting streams, if this executive proposal passes.
+
+* `yank` DAI stream ID 22.
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - May 10, 2023.md:131]: I would reword this, maybe say how much is cancelled? Idk. Probably don't use 'yank'

File: governance/votes/Executive vote - May 10, 2023.md [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+* `yank` MKR stream ID 37.
+
+### Increase MKR_VEST `cap`
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - May 10, 2023.md:134]: Should potentially have a forum post from a dev linked in somewhere?

File: governance/votes/Executive vote - May 10, 2023.md [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+
+* Increase the `cap` to **2,200 MKR**.
+
+This will allow for the full agreed amount of MKR to be streamed to PullUp along with a 10% buffer.
+
+## Review
+
+Community debate on these topics can be found on the MakerDAO [Governance forum](https://forum.makerdao.com/). Please review any linked threads to inform your position before voting.
+
+Additionally, these changes may have been discussed further in recent Governance calls. [Video](https://www.youtube.com/playlist?list=PLLzkWCj8ywWNq5-90-Id6VPSsrk4OWVan) for these calls is available to review.
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - May 10, 2023.md:146]: These don't happen anymore, remove this section.

File: governance/votes/Executive vote - May 10, 2023.md [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+
+---
+
+## Resources
+
+Additional information about the Governance process can be found in the [Maker Operational Manual](https://manual.makerdao.com).
+
+To participate in future Governance calls, please [join us](https://forum.makerdao.com/tag/pubcall-:-governance-and-risk) every Thursday at 17:00 UTC.
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - May 10, 2023.md:154]: Should be removed / replaced. These calls no longer take place.

**Review by LongForWisdom — CHANGES_REQUESTED** [review]
See comments.

File: governance/votes/Executive vote - May 10, 2023.md (hunk 4 above)
Comment (patrick-j-govalpha) [reply governance/votes/Executive vote - May 10, 2023.md:101]: added hours in parentheses

File: governance/votes/Executive vote - May 10, 2023.md (hunk 5 above)
Comment (patrick-j-govalpha) [reply governance/votes/Executive vote - May 10, 2023.md:112]: good suggestion

File: governance/votes/Executive vote - May 10, 2023.md (hunk 6 above)
Comment (patrick-j-govalpha) [reply governance/votes/Executive vote - May 10, 2023.md:132]: added a description

File: governance/votes/Executive vote - May 10, 2023.md (hunk 7 above)
Comment (patrick-j-govalpha) [reply governance/votes/Executive vote - May 10, 2023.md:135]: For this I'm not sure it's strictly necessary, it's (hopefully) fairly self explanatory why we're changing this

File: governance/votes/Executive vote - May 10, 2023.md (hunk 8 above)
Comment (patrick-j-govalpha) [reply governance/votes/Executive vote - May 10, 2023.md:169]: good points - done

File: governance/votes/Executive vote - May 10, 2023.md [hunk 10]
```diff
@@ -97,8 +98,10 @@ As per [this poll](https://vote.makerdao.com/polling/QmYFfRuR) the following par
 
 #### [Ceiling Increase Cooldown (`ttl`)](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl) Changes
 
-* Increase the WSTETH-A `ttl` by 21,600 seconds from 21,600 seconds to **43,200 seconds**.
-* Increase the WSTETH-B `ttl` by 28,800 seconds from 28,800 seconds to **57,600 seconds**.
+* Increase the WSTETH-A `ttl` by 21,600 seconds (6 hours) from 21,600 seconds (6 hours) to **43,200 seconds** (12 hours).
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - May 10, 2023.md:101]: I would honestly just cut the seconds altogether and go with only hours, but whatever you prefer.

File: governance/votes/Executive vote - May 10, 2023.md [hunk 11]
```diff
@@ -108,7 +111,7 @@ As per [this initial poll](https://vote.makerdao.com/polling/QmUnhQZy) and this
 
 ### Vesting Stream Setup
 
-As per [MIP106](https://mips.makerdao.com/mips/details/MIP106), the following DAI and MKR vesting streams will be created if this executive proposal passes:
+As per [MIP106c2.6.6.2.1A](https://mips.makerdao.com/mips/details/MIP106#6-6-2-1a-), the following DAI and MKR vesting streams will be created if this executive proposal passes:
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - May 10, 2023.md:114]: I would probably not put the full dot address in the plaintext. Just the URL.

File: governance/votes/Executive vote - May 10, 2023.md [hunk 12]
```diff
@@ -139,18 +144,36 @@ To facilitate the MKR vesting stream to Phoenix Labs, the `cap` in the MKR_VEST
 
 This will allow for the full agreed amount of MKR to be streamed to PullUp along with a 10% buffer.
 
+### DAO Resolution
+
+As per [this post](https://forum.makerdao.com/t/dao-resolution-to-facilitate-onboarding-of-taco-with-exchange-agent/20735) from the Strategic Finance Core Unit (SF-001), the following DAO Resolution will be approved if this executive proposal passes.
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - May 10, 2023.md:149]: Should this refer to steakhouse instead? I recall we referred to dewiz versus CES elsewhere.

**Review by LongForWisdom — CHANGES_REQUESTED** [review]
See comments
